docs/development/developing.md

```diff
@@ -1,3 +1,56 @@
+# PyScript
+
+## What is PyScript
```

Comment:
I don't think this should be the first thing here.

This page is related to developing, using pyscript as a header will override the development title below (pages should only have one h1 header).

Reply:
oh ok

Reply:
where should i add link to developing page in readme

Reply:
I think it should be under the `How to Contribute` in the Readme 😄